docs/concepts/jcloud/index.md

```diff
@@ -226,6 +226,89 @@ export JCLOUD_NO_INTERACTIVE=1
 ```
 
 
+### Update Flow
+
+You can update a Flow by providing an updated YAML.
+
+To update a Flow:
+
+```bash
+jc update super-mustang-c6cf06bc5b flow.yml
+```
+
+```{figure} img/update_flow.png
+:width: 70%
+```
+
+### Pause / Resume Flow
+
+You have the option to `pause` a Flow that is not currently in use but may be needed later. This will allow the Flow to be resumed later when it is needed again by using `resume`.
```

Comment:
```suggestion
You have the option to pause a Flow that is not currently in use but may be needed later. This will allow the Flow to be resumed later when it is needed again by using `resume`.
```
When using it as a verb (not a command). Don't use backticks. Ensure this is consistent throughout doc.